app/views/concepts/show.html.haml

```diff
@@ -1,19 +1,19 @@
 .section--primary
   .container--offset
-    %h1= @concept.name
+    %h1= sanitize @concept.name
 
 %section#content{ tabindex: "0" }
   .container--offset
-    = MarkdownHelper.markdown(@concept.definition)
+    = MarkdownHelper.markdown(sanitize @concept.definition).html_safe
```

Comment:
Can we stick to standardised use of `sanitize` - you've got it with parentheses in most other places, so should be `= MarkdownHelper.markdown(sanitize(@concept.definition)).html_safe`